CMakeLists.txt

```diff
@@ -50,11 +50,11 @@ if (APPLE)
    set(CMAKE_OSX_ARCHITECTURES "x86_64")
 endif()
 
-if (DEFINED PYTHON_INCLUDE_DIR AND DEFINED PYTHON_LIBRARIES AND DEFINED PYTHON_EXECUTABLE)
+if (DEFINED PYTHON_INCLUDE_DIR AND DEFINED PYTHON_LIBRARIES AND DEFINED Python_EXECUTABLE)
     SET(PYTHON_INCLUDE_DIRS "${PYTHON_INCLUDE_DIR}")
     SET(PYTHONLIBS_FOUND TRUE)
-    find_package(PythonInterp 2.7 REQUIRED)
-    if(NOT PYTHONINTERP_FOUND)
+    find_package(Python 2.7 EXACT REQUIRED COMPONENTS Interpreter)
```

Comment:
I opted for `Python 2.7 EXACT` here to make it explicit that Python 3 is not (yet) supported. This should accept any patch version of 2.7.X though. But I used `Python` rather than `Python2` in hopes of making it easier for when we do want to transition to Python 3. The `Python_EXECUTABLE` and `Python_VERSION` usages shouldn't need to change again.